engine/jsonlint/pipe.json

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "https://repometric.github.io/linterhub/schema/extr.json",
+    "id": "jsonlint",
+    "name": "jsonlint",
+    "pipeline": [
+        {
+            "cmd": "jsonlint",
+            "success": 1,
+            "engine": true,
+            "args":{
+                "--compact":"",
```

Comment:
As for me it will not work in real situation. Here you specify some default values that could not be changed by user. Where to place other values that _were_ specified by user? Before, after, etc?

Reply:
Create issue in `lhexec` by number [`5`](../../lhexec/issues/5)

Reply:
BTW then the value is `null`

Reply:
If argument is a flag that has no value, then in JSON it has null as value.

Reply:
Create issue in `ep` by number [48](../../ep/issues/48)